src/components/rewards-popup/styles.scss

```diff
@@ -45,8 +45,14 @@
 
   &__close-button {
     position: absolute;
-    top: 16px;
-    right: 16px;
+    top: 18px;
+    right: 18px;
```

Comment:
`18px` might look funky from a code-standpoint, but the `IconButton` padding is slightly different in zUI and zDS, so I had to accomodate for this.

Reply:
Yeah I think IconButton needs some updating. I often have to put a container around it to fix up the size correctly as per figma design. Normally in figma the svgs are 24px with a container of 32px, but setting `size={32}` on `IconButton` sets the svg to 32px not the container....if that makes sense.